src/sasctl/pzmm/write_json_files.py

```diff
@@ -950,3 +982,629 @@ def convertDataRole(self, dataRole):
             conversion = 1
 
         return conversion
+
+    @classmethod
+    def create_requirements_json(cls, json_path=Path.cwd()):
+        """
```

Comment:
nitpick: single-line short description followed by an extended summary, per Numpydoc style guide: [https://numpydoc.readthedocs.io/en/latest/format.html#short-summary](https://numpydoc.readthedocs.io/en/latest/format.html#short-summary)

Multiple methods defined in te class have this (minor) problem

Reply:
Done.